tests/integration/WatchmanTestCase.py

```diff
@@ -29,6 +29,13 @@
 def norm_path(name):
     return os.path.normcase(os.path.normpath(name))
 
+def conv_path_to_bytes(path):
+    if (not pywatchman.compat.PYTHON3 and isinstance(path, unicode)) or \
```

Comment:
Also, this would probably be better expressed by just adding `UNICODE` to `pywatchman.compat` or similar.